scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -48,7 +48,7 @@ export function AuthoringTopbarDirective(
                 dispatchInternalEvent('interactiveArticleActionStart', {
                     items: [scope.item],
                     tabs: ['send_to', 'publish'],
-                    activeTab: 'publish',
+                    activeTab: scope.item.flags.marked_for_not_publication ? 'send_to' : 'publish',
```

Comment:
would use probably `item.flags?.marked_...` not sure if we have those flags set always